src/repositories/leaderboard.repository.ts

```
@@ -0,0 +1,155 @@
+
+  // 매핑된 정렬 컬럼으로 ORDER BY 절 및 LIMIT 절 빌드
+  private buildLeaderboardGroupOrderClause(sortCol: string, type: string) {
+    if (type === 'post') {
+      return `
+        ORDER BY ${sortCol} DESC
+        LIMIT $2;
+      `;
+    } else {
+      return `
+        GROUP BY u.id
+        ORDER BY ${sortCol} DESC
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)